samples/notebooks/polyglot/Azure logs.ipynb

```diff
@@ -140,18 +171,100 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": ".NET (PowerShell)",
-   "language": "PowerShell",
-   "name": ".net-powershell"
+   "display_name": ".NET (C#)",
+   "language": "C#",
+   "name": ".net-csharp"
   },
   "language_info": {
    "file_extension": ".ps1",
    "mimetype": "text/x-powershell",
    "name": "PowerShell",
    "pygments_lexer": "powershell",
    "version": "7.0"
+  },
+  "polyglot_notebook": {
+   "kernelInfo": {
+    "defaultKernelName": "powershell",
+    "items": [
+     {
+      "aliases": [],
+      "languageName": null,
+      "name": ".NET"
+     },
+     {
+      "aliases": [
+       "C#",
+       "c#"
+      ],
+      "languageName": "C#",
+      "name": "csharp"
+     },
+     {
+      "aliases": [
+       "F#",
+       "f#"
+      ],
+      "languageName": "F#",
+      "name": "fsharp"
+     },
+     {
+      "aliases": [],
+      "languageName": "HTML",
+      "name": "html"
+     },
+     {
+      "aliases": [
+       "js"
+      ],
+      "languageName": "JavaScript",
+      "name": "javascript"
+     },
+     {
+      "aliases": [],
+      "languageName": "KQL",
+      "name": "kql"
+     },
+     {
+      "aliases": [],
+      "languageName": "Mermaid",
+      "name": "mermaid"
+     },
+     {
+      "aliases": [],
+      "name": "powershell"
+     },
+     {
+      "aliases": [
+       "powershell"
+      ],
+      "languageName": "PowerShell",
+      "name": "pwsh"
+     },
+     {
+      "aliases": [],
+      "languageName": "SQL",
+      "name": "sql"
+     },
+     {
+      "aliases": [],
+      "languageName": null,
+      "name": "value"
+     },
+     {
+      "aliases": [
+       "frontend"
+      ],
+      "languageName": null,
+      "name": "vscode"
```

Comment:
Should this be in the list?

Reply:
it is one of the kernel present

Reply:
filtering out kernels not implementing submitcode